### PR TITLE
Aalind tyagi/robocup sw tutorial

### PR DESCRIPTION
**Pull request by Squid5678**
Did SoccerMom

### CODE REVIEW

**Comment by Squid5678** [comment]
Screenshot 2023-10-01 at 8 17 13 PM
Screenshot 2023-10-01 at 8 17 25 PM

File: llvm.sh.1
Comment (sid-parikh) [inline llvm.sh.1:1]: Hey, please don't commit these extra files to git.

File: llvm.sh.2
Comment (sid-parikh) [inline llvm.sh.2:1]: same comment here

File: soccer/src/soccer/radio/.vscode/c_cpp_properties.json
Comment (sid-parikh) [inline soccer/src/soccer/radio/.vscode/c_cpp_properties.json:1]: and here. you can look up various ways to tell git to ignore files and folders. if you think an ignore is global enough that everyone needs it, we can update our .gitignore file.

**Review by sid-parikh — APPROVED** [review]
This is wonderful work Aalind!

**Pull request closed without merging**